proto/one/operations/sample/analyte_result.proto

```diff
@@ -0,0 +1,60 @@
+syntax = "proto3";
+import "google/protobuf/wrappers.proto";
+import "jsonTicksDateTime.proto";
+import "enum_import_status.proto";
+option csharp_namespace ="ONE.Models.CSharp";
+
+message AnalyteResult {
+
+    // AuthTwinRefId
+    google.protobuf.StringValue authTwinRefId = 1;
+    
+    // The Sample Number (letter and numbers) that describe the sample
+    string sampleNumber = 2;
+    
+    // The name of the analyte.  (The subject of the activity)
+    string analyteName = 3;
+
+    // Analysis Result.  May include qualifier
+    string result = 4;
```

Comment:
Since this may include a qualifier do we also need a numeric value like we have in Spreadsheets and Historian?

Reply:
The web doesn't send / model a string and numeric value here.  If you look at the request, in the property bag, it's just a string result.  I tried this using a qualifier, and it looks like this...